src/main/java/org/edumips64/MainCLI.java

```diff
@@ -62,146 +67,46 @@ public static void main(String args[]) {
         cfg.resetConfiguration();
       }
 
-      // Initialize the CPU and all its dependencies.
-      Memory memory = new Memory();
-      CPU c = new CPU(memory, cfg, new BUBBLE());
-      LocalFileUtils localFileUtils = new LocalFileUtils();
-      SymbolTable symTab = new SymbolTable(memory);
-      IOManager iom = new IOManager(localFileUtils, memory);
-      Dinero dinero = new Dinero();
-      InstructionBuilder instructionBuilder = new InstructionBuilder(memory, iom, c, dinero, cfg);
-      Parser p = new Parser(localFileUtils, symTab, memory, instructionBuilder);
-      c.setStatus(CPU.CPUStatus.READY);
+      Cli cli = new Cli(cfg);
+      commandLine = new CommandLine(cli);
 
+      System.out.println(CurrentLocale.getString("CLI.WELCOME"));
       // Initialization done. Print a welcome message and open the file if needed.
-      System.out.println("Welcome to EduMIPS64 CLI shell!");
       if (cliArgs.getFileName() != null) {
-        String absoluteFilename = new File(cliArgs.getFileName()).getAbsolutePath();
-        try {
-          p.parse(absoluteFilename);
-        } catch (ParserMultiException e) {
-          if (e.hasErrors()) {
-            throw e;
-          }
-        }
-        dinero.setDataOffset(memory.getInstructionsNumber()*4);
-        c.setStatus(CPU.CPUStatus.RUNNING);
-        System.out.println("(Loaded file " + absoluteFilename + ")");
+          commandLine.execute("file", cliArgs.getFileName());
```

Comment:
The command is `load` now, right?

```suggestion
          commandLine.execute("load", cliArgs.getFileName());
```